app/server/appsmith-interfaces/src/main/java/com/appsmith/external/constants/spans/ce/LayoutSpanCE.java

```diff
@@ -15,4 +15,14 @@ public class LayoutSpanCE {
             APPSMITH_SPAN_PREFIX + "onLoadExecutablesUtil.updateExecutablesExecuteOnLoad";
     public static final String FIND_AND_UPDATE_LAYOUT =
             APPSMITH_SPAN_PREFIX + "onLoadExecutablesUtil.findAndUpdateLayout";
+
+    public static final String EXTACT_AND_SET_EXECUTABLE_BINDINGS_IN_GRAPH_EDGES =
```

Comment:
Can we if the typo here 
```suggestion
    public static final String EXTRACT_AND_SET_EXECUTABLE_BINDINGS_IN_GRAPH_EDGES =
```